src/i_log.h

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright(C) 2016 Adar Arnon
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+//
+// DESCRIPTION:
+//     Log interface, to avoid using printf() etc. in the code.
+//
+
+#ifndef __I_LOG__
+#define __I_LOG__
+
+#include <stdarg.h>
+
+// Initialize the log
+void I_LogInit(void);
+
+// Write an info log message
+void I_LogInfo(const char *fmt, ...);
```

Comment:
Presumably you're planning to replace every call to printf() with this function. We should bear in mind that this is going to be used throughout the codebase a lot. With that in mind, I think it would be worth choosing names to that effect, for example:

`I_LogInfo` -> `I_Printf`
`I_LogInfoV` -> `I_VPrintf`
`I_LogError` -> `I_ErrorPrintf`
`I_LogErrorV` -> `I_VErrorPrintf`

Just suggestions though, maybe you have better suggestions.